tools/dev/config.yaml

```diff
@@ -33,14 +33,25 @@ environments:
     name: janusgraph
     database_config:
       url: http://localhost:8182
+      analytics_engine:
+        enabled: true
+        type: spark-analytics-api
+        url: http://localhost:8100
+  analytics_engine:
+    enabled: true
+    defaultUseAnalyticsEngine: false
```

Comment:
@bobvanluijt I don't quite understand the changes in this file. Why do you turn on an analytics engine here? We don't spin up Spark in our every day development setup?

Reply:
This was a wrong assumption, I've removed it.

Reply:
This assumption was copied from the other ones. What would be the correct settings for this file?